@kiva/kv-components/vue/KvEditButton.vue

```diff
@@ -1,104 +1,30 @@
 <template>
-	<div>
-		<!-- Edit Button -->
-		<KvButton
-			variant="secondary"
-			aria-label="Edit"
-			class="tw-w-7 tw-h-10"
-			@click="showLightbox"
-		>
-			<KvMaterialIcon
-				class="tw-w-3 tw-h-3"
-				:icon="mdiPencil"
-			/>
-		</KvButton>
-
-		<!-- Lightbox -->
-		<KvLightbox
-			:visible="isVisible"
-			:title="title"
-			:prevent-close="preventClose"
-			:variant="variant"
-			@lightbox-closed="hideLightbox"
-		>
-			<template #header>
-				<slot name="header"></slot>
-			</template>
-
-			<!-- default slot (body) -->
-			<slot></slot>
-
-			<!-- controls slot -->
-			<template #controls>
-				<slot
-					name="controls"
-					:hideLightbox="hideLightbox"
-				></slot>
-			</template>
-		</KvLightbox>
-	</div>
+	<!-- Edit Button -->
+	<KvButton
```

Comment:
Will this component get more functionality in the near future? If not, I'm not sure we need this as a shared component in our UI library. It's currently just a button with an icon.

Reply:
Okay, no storybook component needed for this then.